go.mod

```diff
@@ -55,6 +55,7 @@ require (
 	github.com/spaolacci/murmur3 v0.0.0-20180118202830-f09979ecbc72
 	github.com/struCoder/pidusage v0.1.2
 	github.com/tiancaiamao/appdash v0.0.0-20181126055449-889f96f722a2
+	github.com/tiancaiamao/debugger v0.0.0-20190426085344-91156e93f38e
```

Comment:
Can we put this package into `failpoint`?

Reply:
https://github.com/pingcap/tidb/pull/10278#discussion_r279179951